share/nisar/defaults/gcov.yaml

```
@@ -191,8 +191,10 @@ runconfig:
 
                 # Flag indicating whether the output GCOV product should include
                 # off-diagonal covariance elements. If False, only the diagonal
-                # elements are included.
-                fullcovariance:   False
+                # elements are included. If empty, `fullcovariance` is set to
+                # `True` any frequency to be processed includes a full-pol dataset
+                # (3 or 4 polarizations), and `False` otherwise
+                fullcovariance: None
```

Let's just leave this empty instead of setting the default to the literal `"None"` so that the yaml parser treats it as the Python object `None` instead of the string `"None"`.

This is also consistent with how we treat other similar parameters in the runconfig that default to None.

```suggestion
                fullcovariance:
```